internal/agentcfg/elasticsearch_test.go

```diff
@@ -79,6 +80,9 @@ func newElasticsearchFetcher(
 	i := 0
 
 	fetcher := NewElasticsearchFetcher(newMockElasticsearchClient(t, func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodDelete && strings.HasPrefix(r.URL.Path, "/_search/scroll") {
+			return
```

Comment:
nit: Can you check that the scroll id is `respTmpl["_scroll_id"]`? We had a bug with scrolling in the past